### PR TITLE
expose selfVideo in VoiceState

### PR DESCRIPTION
**Pull request by Khaaz**
discord/discord-api-docs#1729

### CODE REVIEW

**Comment by cyclane** [comment]
You should also implement this in `index.d.ts` as well as add selfVideo on OldVoiceState in the `voiceStateUpdatee` event

**Comment by bsian03** [comment]
Khaaz intentionally left out typings as a result of #993.. I'm sure he mentioned that a few times already
That being said he also mentioned that you can PR on his fork to add them

File: index.d.ts [hunk 1]
```diff
@@ -419,7 +419,8 @@ declare namespace Eris {
     mute: boolean;
     selfDeaf: boolean;
     selfMute: boolean;
-    selfStream: boolean;
+	selfStream: boolean;
+	selfVideo: boolean;
```
Comment (bsian03) [inline index.d.ts]:
```suggestion
    selfStream: boolean;
    selfVideo: boolean;
```

File: index.d.ts [hunk 2]
```diff
@@ -2423,7 +2424,8 @@ declare namespace Eris {
...
```
Comment (bsian03) [inline index.d.ts]:
```suggestion
    selfStream: boolean;
    selfVideo: boolean;
```

**Review by bsian03 — CHANGES_REQUESTED** [review]
Indents

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**